src/pages/home/slider-home-page/slider-home-page.js

```diff
@@ -1,38 +1,41 @@
 import React from 'react';
 import AwesomeSlider from 'react-awesome-slider';
 import 'react-awesome-slider/dist/styles.css';
-import { useSelector } from 'react-redux';
+import { useStore } from 'react-redux';
 import { useStyles } from './slider-home-page.style';
-import { getImage } from '../../../utils/imageLoad';
-import { Backdrop } from '@material-ui/core';
-import LoadingBar from '../../../components/loading-bar';
+import withAutoplay from 'react-awesome-slider/dist/autoplay';
+import { carouselInterval } from '../../../configs';
+
+const AutoplaySlider = withAutoplay(AwesomeSlider);
 
 const SliderHomePage = () => {
+  const store = useStore();
   const styles = useStyles();
-  const { models, loading } = useSelector(({ Model }) => ({
-    models: Model.models,
-    loading: Model.loading
-  }));
-  if (loading) {
-    return (
-      <Backdrop className={styles.backdrop} open={loading} invisible>
-        <LoadingBar color='inherit' />
-      </Backdrop>
-    );
+
+  function getImages(state) {
+    return state.Categories;
+  }
+  let curentValue = getImages(store.getState());
+  let images = [];
+  for (let i = 0; i < curentValue.list.length; i++) {
```

Comment:
After changing the data selection approach use the `map` method inside the component `return` statement for displaying the array of images

Reply:
Fixed